tests/x11/hexchat_ssl.pm

```diff
@@ -0,0 +1,56 @@
+# SUSE's openQA tests - FIPS tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Case #1459498 - FIPS : hexchat_ssl
+
+use base "x11test";
+use strict;
+use testapi;
+
+sub run() {
+    my $name = ('hexchat');
+    ensure_installed($name);
+    # we need to move the mouse in the top left corner as hexchat
+    # opens it's window where the mouse is. mouse_hide() would move
+    # it to the lower right where the pk-update-icon's passive popup
+    # may suddenly cover parts of the dialog ... o_O
+    mouse_set(0, 0);
+
+    if (my $url = get_var("XCHAT_URL")) {
+        x11_start_program("$name --url=$url");
+    }
+    else {
+        x11_start_program("$name");
+        assert_screen "$name-network-select";
+        type_string "freenode\n";
+
+        # use ssl for all servers on this network
+        assert_and_click "$name-edit-button";
+        assert_and_click "$name-use-ssl-button";
+        assert_and_click "$name-close-button";
+
+        assert_and_click "$name-connect-button";
+        assert_screen "$name-connection-complete-dialog";
+        assert_and_click "$name-join-channel";
+
+        send_key "ctrl-a";
+        send_key "delete";
+        type_string "#openqa\n";
```

Comment:
It might be a funny idea to join such a channel but maybe it should be a bit more "private", e.g. #openqa-test_irc_from_openqa
What do others think?